molecule/scenario_resources/verify.yml

```diff
@@ -107,6 +107,40 @@
       when: item.rc != 0
       with_items: "{{ test_results.results }}"
 
+    - name: Obtain 1 node's database fields encryption key
+      slurp:
```

Comment:
cool!

Reply:
Yeah, it stores it in memory as base64 encoded.